tensorflow/compiler/xla/service/gpu/gpu_performance_model.cc

```diff
@@ -37,6 +37,8 @@ static constexpr float kL1CacheSpeedup = 8;
 // much smaller than the cache size will likely stay in it.
 // For reference, it can be up to 256 kB per SM on RTX A6000.
 static constexpr float kL1CacheSizePerSM = 2 * 1024;
+// kL1CacheSizePerSM for MI300
+static constexpr float kL1CacheSizePerSM_MI300 = 32 * 1024;
```

Comment:
maybe rename it as `kL1CacheSizePerSM_GFX942` ? because it's for MI300 and MI308